uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/utils/DateUtils.kt

```diff
@@ -178,6 +179,26 @@ abstract class DateUtils {
             return getWeekdayNames(GregorianCalendar.SHORT, firstWeekday)
         }
 
+        /**
+         * Returns a vector of Int representing the frequency of each weekday in a given month.
+         *
+         * @param startOfMonth a Timestamp representing the beginning of the month.
+         */
+        @JvmStatic
+        fun getWeekdaysInMonth(startOfMonth: Timestamp): Array<Int> {
+            val month = startOfMonth.toCalendar()[Calendar.MONTH] + 1
```

Comment:
Why `+1`?

Reply:
The Calendar class encodes the months with numbers from 0 to 11, but the YearMonth.of(year, month) method used below to get the length of the month expects an int from 1 to 12 for the month, so I just add 1 to use that method correctly.